hs/Scripts.hs

```diff
@@ -11,39 +11,39 @@ import MasterTemplate
 
 plannerScripts :: H.Html
 plannerScripts = concatHtml (map makeScript["http://code.jquery.com/jquery-1.10.2.js",
-                                           "http://code.jquery.com/ui/1.10.4/jquery-ui.js",
-                                           "static/js/graph/modal.js",
-                                           "static/js/graph/objects/edge.js",
-                                           "static/js/graph/objects/node.js",
-                                           "static/js/common/objects/course.js",
-                                           "static/js/common/facebook_login.js",
-                                           "static/js/common/cookieHandler.js",
-                                           "static/js/graph/tabs/setup_tabs.js",
-                                           "static/js/graph/utilities/course_description.js",
-                                           "static/js/graph/tabs/feedback_form.js",
-                                           "static/js/graph/tabs/focuses.js",
-                                           "static/js/graph/tabs/post.js",
-                                           "static/js/graph/tabs/timetable.js",
-                                           "static/js/graph/tabs/fce_count.js",
-                                           "static/js/common/objects/section.js",
-                                           "static/js/common/utilities/util.js",
-                                           "static/js/graph/utilities/structs.js",
-                                           "static/js/graph/utilities/util.js",
-                                           "static/js/graph/create_data.js",
-                                           "static/js/graph/parse_graph.js",
-                                           "static/js/graph/mouse_events.js",
-                                           "static/js/graph/setup.js"])
+                                            "http://code.jquery.com/ui/1.10.4/jquery-ui.js",
+                                            "http://localhost:8000/static/js/graph/modal.js",
+                                            "http://localhost:8000/static/js/graph/objects/edge.js",
```

Comment:
Actually, even easier: Run the server on your happstack branch, and go to `http://localhost:8000/graph/`- there should be a surprise waiting there.